packages/in_app_purchase/in_app_purchase_platform_interface/lib/src/types/product_details.dart

```diff
@@ -42,4 +43,7 @@ class ProductDetails {
   /// The currency code for the price of the product.
   /// Based on the price specified in the App Store Connect or Sku in Google Play console based on the platform.
   final String currencyCode;
+
+  ///The currency symbol for the locale, e.g. $ for US locale.
```

Comment:
Turns out in some cases (applies mainly on Android) the currency symbol cannot be determined. The current Java tooling used (https://docs.oracle.com/javase/7/docs/api/java/util/Currency.html#getSymbol()) returns the ISO 4217 currency code when the symbol cannot be determined. It would be good to mention this in the documentation.

Also a small nit, the documentation is missing a space after the `///`.

Maybe we can change the documentation to something like below. Of course this means that we need to ensure this will work on all platforms in the same way.
```suggestion
  /// The currency symbol for the locale, e.g. $ for US locale.
  ///
  /// When the currency symbol cannot be determined, the ISO 4217 currency code is returned. 
```

Reply:
@mvanbeusekom On iOS I can't find what it does 'if not found', maybe it just supports all currency symbols?
https://developer.apple.com/documentation/foundation/nslocale/1642814-currencysymbol

Reply:
Maybe we can add a simple check, if the `currencySymbol` is `nil` or empty we replace it with the `currencyCode`?